.github/workflows/pipeline_kind_test.yaml

```diff
@@ -30,5 +30,7 @@ jobs:
       run: |
         cd apps/pipeline/upstream
         kubectl create ns kubeflow
+        kubectl apply -f third-party/metacontroller/base/crd.yaml
+        kubectl wait --for condition=established crd compositecontrollers.metacontroller.k8s.io
```

Comment:
In addition to syncing pipelines manifest, I had to add additional commands in the github workflow for pipeline test to wait for the CRD creation before creating the `CompositeController`. Without it, it runs into the [no matches for kind "CompositeController" in version "metacontroller.k8s.io/v1alpha1" error](https://github.com/kubeflow/manifests/runs/7329985403?check_suite_focus=true)

Reply:
Can you help point out how this CRD is installed when using this? https://github.com/kubeflow/manifests/blob/master/example/kustomization.yaml

Reply:
Hi Suraj, I believe it comes from here https://github.com/kubeflow/manifests/blob/847faeb5533b5f76423f756c75a41452f302929f/example/kustomization.yaml#L29 which leads to https://github.com/kubeflow/manifests/blob/847faeb5533b5f76423f756c75a41452f302929f/apps/pipeline/upstream/env/platform-agnostic-multi-user/kustomization.yaml#L13

Reply:
Hi Anna, thanks. I missed the `cd apps/pipeline/upstream` in the script before when I posted the comment but forgot to resolve it afterwards.